src/OceanSimulations/OceanSimulations.jl

```diff
@@ -29,6 +30,10 @@ default_or_override(default::Default, value=default.value) = value
 # Some defaults
 default_free_surface(grid) = SplitExplicitFreeSurface(grid; cfl=0.7)
 
+# 70 substeps is a safe rule of thumb for an ocean at 1/4 - 1/10th of a degree
+# TODO: pass the cfl and a given Δt to calculate the number of substeps?
+default_free_surface(grid::TripolarGrid) = SplitExplicitFreeSurface(grid; substeps = 70)
```

Comment:
```suggestion
# 70 substeps is a safe rule of thumb for an ocean at 1/4 - 1/10th of a degree
# TODO: pass the cfl and a given Δt to calculate the number of substeps?
default_free_surface(grid::TripolarGrid) = SplitExplicitFreeSurface(grid; substeps = 70)
```

Why not explicitly encode the resolution dependence

Reply:
IT doesn't matter if the numbers aren't quite right, it will just help us have a scaffolding to evolve the defaults, eg 

```julia
function default_free_surface(grid::TripolarGrid)
    # find minimum dx and dy
    # compute substeps as function of resolution
    # build free surface
end
```

Reply:
or better yet I guess you can compute the CFL at every grid point since it co-varies with depth and grid spacing